### PR TITLE
Configure OpenAPI as part of webapi project template creation

### PR DESCRIPTION
**Pull request by pranavkm**
Fixes https://github.com/dotnet/aspnetcore/issues/25080

### CODE REVIEW

File: src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-CSharp/.template.config/template.json [hunk 1]
```diff
@@ -273,6 +273,12 @@
     "GenerateApiOrGraph": {
         "type": "computed",
         "value": "(GenerateApi || GenerateGraph)"
+    },
+    "EnableOpenAPI": {
+      "type": "parameter",
+      "datatype": "bool",
+      "defaultValue": "false",
```
Comment (dougbu) [inline src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-CSharp/.template.config/template.json:280]: Is defaulting to `false` what we decided on @mkArtakMSFT and @bradygaster❔

**Review by dougbu — APPROVED** [review]
Fine by me but wait for @bradygaster

**Comment by dougbu** [comment]
Whoa that's a lot of Windows 10 failures but I can't see how they connect to these changes

**Review by javiercn — APPROVED** [review]

**Review by phenning — APPROVED** [review]
Host and symbol changes LGTM

**Pull request merged**